app/views/layouts/application.html.erb

```diff
@@ -16,6 +16,12 @@
     <link rel="apple-touch-icon-precomposed" href="/images/pl-255.png">
     <link rel="apple-touch-icon-precomposed apple-touch-icon" href="/images/pl-255.png">
     <link rel="shortcut icon" href="/images/pl-255.png" />
+    <script type="text/javascript" async
+      src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
+    </script>
+    <script type="text/x-mathjax-config">
+      MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});
```

Comment:
Hello @publiclab/reviewers - does this seem OK to include on every page, and will it cause any trouble? Just trying to think broadly here -- does it run on all browsers, and are there any other failure or optimization issues we need to consider before merging this?

Thank you!